packages/app/src/containers/ExportInspector.tsx

```diff
@@ -45,26 +50,117 @@ export default memo(function ExportInspector() {
   const exportFormats = selectedLayer.exportOptions.exportFormats;
 
   // TODO: Handle export formats
+  const handleFileFormat = useCallback(
+    async (exportFormat: Sketch.ExportFormat) => {
+      const {
+        scale,
+        absoluteSize,
+        fileFormat,
+        visibleScaleType,
+      } = exportFormat;
+
+      const adjustSize = {
```

Comment:
This size logic looks good, but let's move it to a separate function and use a `switch`, so it's clearer how width and height are related. E.g.

```ts
function getExportSize(exportFormat: Sketch.ExportFormat, size: Size) {
  switch (visibleScaleType) {
    case Sketch.VisibleScaleType.Width:
      return { width: ..., height: ...}
    case ...
  }
}
```

```ts
const size = getExportSize(exportFormat, selectedLayer.frame)
```